app/controllers/publish/publish_controller.rb

```diff
@@ -15,7 +15,7 @@ def provider
     end
 
     def recruitment_cycle
-      cycle_year = params[:recruitment_cycle_year] || Settings.current_recruitment_cycle_year
+      cycle_year = params[:recruitment_cycle_year] || params[:year] || Settings.current_recruitment_cycle_year
```

Comment:
At what point does `current_recruitment_cycle_year` change from `2022` to `2023`?

Reply:
Just when rollover ends we update the year constraint